Feature/match request (#9)
adzerk/audience-ios-objc-sdk: README.md

```

If you had any problem with importing the Objective-C class into Swift, please take a look at Apple's guide on [Importing Objective-C into Swift](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift).

### Make a match

> You can match the user's Advertising Identifier with other known identifiers (like an internal ID) so that any event made by the user in the application can be traced back to a known user in the system.

Reviewer: Seems more intuitive to me to explain match as a merge.

```suggestion
You can match the user's Advertising Identifier with other known identifiers (like an internal ID) so that any event made by the user in the application can be merged to a known user in the system.
```

Author: Please check d06f363

__Swift__
```swift
@IBAction func sendMatchEvent(_ sender: Any) {
    let userId1 = VSDKUserId(userId: "bar", "fooType")
    let userId2 = VSDKUserId(userId: "baz", "fooType")
    let idsArray = NSMutableArray(array: [userId1, userId2])
      
    VSDKVelocidi.sharedInstance().match("1234-providerId-56789", userIds: idsArray, onSuccess:{ (response: URLResponse, responseObject: Any) in
        print("Success! Response: \(response)")
    }, onFailure:{(error: Error) in
        print("Failed! Error: \(error.localizedDescription)")
    })
}
```

__Objective-C__
```objc
- (IBAction)sendMatch:(id)sender {

>     VSDKUserId * userId1 =  [[VSDKUserId alloc] initUserId:@"bar":@"foo"];

Reviewer: Have the same type as the Swift example.

```suggestion
    VSDKUserId * userId1 =  [[VSDKUserId alloc] initUserId:@"bar":@"fooType"];
```

>     VSDKUserId * userId2 =  [[VSDKUserId alloc] initUserId:@"baz":@"foo"];

Reviewer: Have the same type as the Swift example.

```suggestion
    VSDKUserId * userId2 =  [[VSDKUserId alloc] initUserId:@"baz":@"fooType"];
```
